python/docs/source/index.md

```diff
@@ -3,94 +3,119 @@ hide-toc: true
+Kaskada is seamlessly integrated with Python's ecosystem of AI/ML tooling so you can load data, process it, train and serve models all in the same place.
 
-Connect to existing data in streams or databases, or load data using Python.
-Wherever your events are stored, Kaskada can help you process them.
+There's no infrastructure to provision (and no JVM hiding under the covers), so you can jump right in - check out the [Quick Start](quickstart).
 
-Build temporal queries and process the results using Python.
-Connect straight to your visualizations, dashboards or machine learning systems.
 
-Kaskada lets you do it all in one place.
+(scalable-and-reliable)=
+### Built for scale and reliability
 
-(get_started)=
-# Get Started
-
-With no infrastructure to deploy, get started processing events immediately.
-Check out the [Quick Start](quickstart) now!
+Implemented in [Rust](https://www.rust-lang.org/) using [Apache Arrow](https://arrow.apache.org/), Kaskada's compute engine uses columnar data to efficiently execute large historic or streaming queries.
+Every operation in Kaskada is implemented incrementally, allowing automatic recovery if the process is terminated or killed.
 
-(local-and-distributed)=
-# Local, Remote and Distributed
+Kaskada makes most jobs fast enough to run locally, so it's easy to build and test your real-time queries.
+As your needs grow, Kaskada's cloud-native design and support for partitioned exeuction gives you the volume and throughput you need to scale.
+Kaskada was built by core contributors to [Apache Beam](https://beam.apache.org/), [Google Cloud Dataflow](https://cloud.google.com/dataflow), and [Apache Cassandra](https://cassandra.apache.org/), and is under active development
 
-Fast enough to run locally, Kaskada makes it easy to build and test your real-time queries.
+* * *
 
-Built for the cloud and supporting partitioned and distributed execution, Kaskada scales to the volume and throughput you need.
+(example-app)=
+## Example Real-Time App: BeepGPT
+
+[BeepGPT](https://github.com/kaskada-ai/beep-gpt/tree/main) keeps you in the loop without disturbing your focus. Its personalized, intelligent AI continuously monitors your Slack workspace, alerting you to important conversations and freeing you to concentrate on what’s most important.
+
+The core of BeepGPT's real-time processing requires only a few lines of code using Kaskada:
+
+```python
+import kaskada as kd
+kd.init_session()
+
+# Bootstrap from historical data
+messages = kd.sources.PyList(
+    rows = pyarrow.parquet.read_table("./messages.parquet")
+        .to_pylist(),
+    time_column_name = "ts", 
+    key_column_name = "channel",
+)
+
+# Send each Slack message to Kaskada
+def handle_message(client, req):
+    messages.add_rows(req.payload["event"])
+slack.socket_mode_request_listeners.append(handle_message)
+slack.connect()
+
+# Aggregate multiple messages into a "conversation"
+conversations = messages \
```

Comment:
You can use parentheses if you want to omit the slashes:

```
conversations = (messages
  .select("user", "text")
  .collect(max=20)
)
```

Reply:
Works for me - we should probably standardize on one way or another